src/OrallyMulticall.sol

```diff
@@ -46,7 +52,7 @@ contract OrallyMulticall is OrallyConsumer {
         return returnData;
     }
 
-    function multitransfer(Transfer[] calldata transfers) public payable onlyExecutor {
+    function multitransfer(Transfer[] calldata transfers) external payable onlyExecutor {
```

Comment:
can you please specify why this changed? cause it'll require do redeploy some infra contracts on already deployed chains

Reply:
Due to [this](https://ethereum.stackexchange.com/questions/19380/external-vs-public-best-practices) answer on stackexchange. In two words, in external functions solidity doesn't need to copy arguments from calldata to memory, as this func won't be called from other func inside this contract.